lib/merge-properties.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+/**
+ * Shallowly merges each argument's own properties into a new object
+ * Does not modify source objects
+ * @param {...object} - the objects to be merged
+ * @returns {object}  - the new object
+ */
+module.exports = function() {
+  var target = {};
+
+  for (var index = 0; index < arguments.length; index++) {
+    var source = arguments[index];
+    for (var key in source) {
```

Comment:
@neurosnap, from what I understand, `Object.keys` performance is not as good as `for...in`.  I'm working on some [benchmarking](https://github.com/scott113341/csjs-benchmarks) for csjs, but I believe that 5a1434c is as fast as this function can be.  Please correct me if I'm wrong!